.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,3 +19,7 @@ Resolves #[relevant GitHub issues, eg 76]
 #### Documentation
 - [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
 - [ ] This PR does not require updating any documentation
+
+### Conjur API update issue
+_Is this PR linked to a Conjur issue about an API update/change?_
+Related [Conjur Issue]( )
```

Comment:
I'd actually move this up as part of the "What ticket does this PR close"
just add an optional line `Relates to cyberark/conjur#ISSUE_NO` under `Resolves #...`, and keep it as simple as that